src/components/src/OrderTableLineItem/ordertable.lineitem.tsx

```diff
@@ -105,7 +105,15 @@ function OrderTableLineItem(props: OrderTableLineItemProps) {
   return (
     <tr className="order-lineitem-row">
       <td className="thumbnail-col">
-        <ImageContainer className="thumbnail" isSkuImage fileName={code} imgUrl={Config.skuImagesUrl.replace('%sku%', code)} />
+        <ImageContainer
+          alt="order line item image"
+          pictureClassName="thumbnail"
+          imgClassName="thumbnail"
+          isSkuImage
+          fileName={code}
+          imageFileTypes={['webp', 'jp2', 'jpg']}
```

Comment:
Would it be worth pulling this out before merging this in?

Reply:
Yep removing these types.  Will be an optional parameter that will override the types in the config.